src/components/filter/Filter.tsx

```diff
@@ -91,59 +64,82 @@ export const Filter = ({ filterTypes, initialFilters, setQueryVariables }: Props
         />
       </TouchableOpacity>
 
-      <Collapsible collapsed={isCollapsed}>
-        <WrapperVertical style={styles.noPaddingTop}>
-          {filterTypes.map((item) => (
-            <WrapperVertical key={item.name} style={styles.noPaddingBottom}>
-              {item.type === FILTER_TYPES.DATE && (
-                <DateFilter
-                  filters={filters}
-                  setFilters={setFilters}
-                  containerStyle={{ width: device.width * 0.45 }}
-                  {...item}
-                  data={item.data as { name: keyof FilterProps; placeholder: string }[]}
-                />
-              )}
-
-              {item.type === FILTER_TYPES.DROPDOWN && item.data?.length && (
-                <DropdownFilter
-                  filters={filters}
-                  setFilters={setFilters}
-                  {...item}
-                  data={item.data as DropdownProps[]}
-                />
-              )}
-
-              {item.type === FILTER_TYPES.SUE.STATUS && (
-                <StatusFilter
-                  filters={filters}
-                  setFilters={setFilters}
-                  {...item}
-                  data={item.data as StatusProps[]}
-                />
-              )}
-            </WrapperVertical>
-          ))}
-        </WrapperVertical>
-
-        <WrapperVertical style={styles.noPaddingBottom}>
-          <Button
-            invert
-            disabled={isNoFilterSet}
-            title={texts.filter.resetFilter}
-            onPress={() => {
-              setIsCollapsed(true);
-
-              setTimeout(() => {
-                setFilters(initialFilters);
-              }, 500);
+      {isOverlay ? (
+        <Modal
+          animationType="slide"
+          onRequestClose={() => setIsCollapsed(!isCollapsed)}
+          presentationStyle="pageSheet"
+          visible={!isCollapsed}
+        >
+          <Header
+            backgroundColor={colors.transparent}
+            centerComponent={{
+              text: texts.filter.filter,
+              style: { color: colors.darkText, fontSize: normalize(18), fontWeight: '700' }
+            }}
+            rightComponent={{
+              icon: 'close',
+              color: colors.darkText,
+              onPress: () => setIsCollapsed(!isCollapsed)
```

Comment:
what about Androids look?

Reply:
I think it looks good.

<img width="270" alt="image" src="https://github.com/user-attachments/assets/0babf15f-b048-4bef-9f42-7e24ef182b18">